samples/javascript_nodejs/84.bot-authentication-certificate/README.md

```diff
@@ -13,51 +17,174 @@ This bot has been created using [Bot Framework](https://dev.botframework.com/),
-```
+    ```bash
+    git clone https://github.com/microsoft/botbuilder-samples.git
+    ```
 
-- In a terminal, navigate to `samples/javascript_nodejs/84.bot-authentication-certificate`
+- Create and configure the SSL/TSL certificate. In this sample we use two possible options to create and set an SSL/TSL certificate. Below is a step-by-step description of each one:
 
-```bash
-cd samples/javascript_nodejs/84.bot-authentication-certificate
-```
+  ### Using local environment
+  - This option requires the following app settings variables:
 
-- Install modules
+    - MicrosoftAppId: App Id of your bot.
 
-```bash
-npm install
-```
+    - CertificateThumbprint: Thumbprint of the certificate uploaded to the app registration.
 
-- Start the bot
+  1. Intall and configure [OpenSSL](https://www.openssl.org/source/) with the latest version
+    - Download the latest version source and add the folder to the [environment variables](https://www.java.com/en/download/help/path.html) path.
+      ```bash
+      setx path "%path%;<OpenSSL path here> 
+      i.e
+      setx path "%path%;C:\Program Files\openssl-3.3.0"
+      ```
 
-```bash
-npm start
-```
+  2. Run the following command in PowerShell
+       - For global environment certificate(Use admin PowerShell) execute:
 
-## Testing the bot using Bot Framework Emulator
+      ```
+      $cert = New-SelfSignedCertificate -CertStoreLocation "." -Subject "CN=<certificate-name>" -KeySpec KeyExchange
+      ```
 
-[Bot Framework Emulator](https://github.com/microsoft/botframework-emulator) is a desktop application that allows bot developers to test and debug their bots on localhost or running remotely through a tunnel.
+      ![Global Certificate Command](Images/Local/GlobalCertificateCommand.png)
 
-- Install the latest Bot Framework Emulator from [here](https://github.com/Microsoft/BotFramework-Emulator/releases)
+       - For current user certificate execute:
 
-### Connect to the bot using Bot Framework Emulator
+      ```
+      $cert = New-SelfSignedCertificate -CertStoreLocation "Cert:\CurrentUser\My" -Subject "CN=<certificate-name>" -KeySpec KeyExchange
+      ```
 
-- Launch Bot Framework Emulator
-- File -> Open Bot
-- Enter a Bot URL of `http://localhost:3978/api/messages`
+      ![User Certificate Command](Images/Local/UserCertificateCommand.png)
 
-## Interacting with the bot
+  3. Then, type _Manage computer certificates(global environment certificate)_ or _Manage User Certificates(current user certificate)_ in the Windows search bar and hit enter.
 
-This sample uses the bot authentication capabilities of Azure Bot Service, providing features to make it easier to develop a bot that authenticates users using digital security certificates. You just need to provide the certificate data linked to the managed identity and run the bot, then communicate with it to validate its correct authentication.
+      ![User Certificate Search](Images/Local/CertificateSearch.png)
```

Comment:
If repo is cloned in linux/unix system, then `Images` is not the same as `images`.